eo-maven-plugin/src/test/java/org/eolang/maven/LogFormatTest.java

```diff
@@ -99,22 +88,24 @@ private static final class MockAppender extends WriterAppender {
          */
         private MockAppender(final Appender console) {
             this.console = console;
-            this.last = new AtomicReference<>();
+            this.events = new ConcurrentLinkedDeque<>();
```

Comment:
@volodya-lombrozo What is the purpose of deqeue here? Can queue fit?

Reply:
@mximp Yes, of corse. Thank you!

Reply:
Done